src/build/__main__.py

```diff
@@ -150,6 +157,10 @@ def main(cli_args, prog=None):  # type: (List[str], Optional[str]) -> None
         parser.prog = prog
     args = parser.parse_args(cli_args)
 
+    if args.version:
```

Comment:
Oh, no need of this cryptic solution, just use the `version` action type from https://docs.python.org/3/library/argparse.html#action